dom/lists/DOMTokenList-supports.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>DOMTokenList.supports</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+"use strict";
+
+var tokens = {
+  // https://html.spec.whatwg.org/multipage/interaction.html#the-dropzone-attribute:concept-supported-tokens
+  el_dropzone: [
+    "copy", "move", "link", "string:test1", "string:test2", "file:test1", "file:test2"
+  ],
+  // https://html.spec.whatwg.org/multipage/embedded-content.html#the-iframe-element:concept-supported-tokens
+  iframe_sandbox: [
+    "allow-forms", "allow-modals", "allow-pointer-lock", "allow-popups",
+    "allow-popups-to-escape-sandbox", "allow-same-origin", "allow-scripts", "allow-top-navigation"
+  ],
+  // https://html.spec.whatwg.org/multipage/semantics.html#links-created-by-a-and-area-elements:concept-supported-tokens
+  a_area_rel: [
+    "noreferrer", "noopener", "prefetch"
+  ],
+  // https://html.spec.whatwg.org/multipage/semantics.html#the-link-element:concept-supported-tokens
+  link_rel: [
+    "alternate", "icon", "pingback", "prefetch", "stylesheet", "next"
+  ],
+  // https://html.spec.whatwg.org/multipage/semantics.html#linkTypes
+  other_rel_values: [
+    "author", "bookmark", "external", "help", "license", "nofollow", "prev", "search", "tag"
+  ],
+  fiction_values: ["", " ", "test"]
+}
+
+var pairs = {
+  // Defined in DOM
+  "classList": {
+    "anyElement" : []
+  },
+  // Defined in HTML
+  "dropzone": {
+    "anyHTMLElement": tokens.el_dropzone
+  },
+  "headers": {
+    "td": [],
+    "th": [],
+  },
+  "htmlFor": {
+    "output": []
+  },
+  "ping": {
+    "a": [],
+    "area": [],
+  },
+  "relList": {
+    "a": tokens.a_area_rel,
+    "area": tokens.a_area_rel,
+    "link": tokens.link_rel
+  },
+  "sandbox": {
+    "iframe": tokens.iframe_sandbox
+  }
+}
+
+// Produce mixed tokens (including uppercase)
+function prepareTokens(el, attr) {
+  var output = [];
+  var mixTonens = [].concat(pairs[attr][el]); // firstly we always check correct tokens
+  Object.keys(tokens).forEach(function(token){ // and some others for comparison
+    mixTonens = mixTonens.concat(tokens[token]);
```

Comment:
I can reduce but, for example, Blink is case sensitive here (per spec is wrong), so minimum is also use uppercase for correct tokens. And other_rel_values also worth checking for a/area/link so that why I make mix.